pyenv-lazy.plugin.zsh

```diff
@@ -6,7 +6,9 @@ fi
 
 # Lazy load pyenv
 if type pyenv > /dev/null; then
-    export PATH="${PYENV_ROOT}/bin:${PYENV_ROOT}/shims:${PATH}"
+    if [[ ! -n "${VIRTUAL_ENV}" ]]; then
+        export PATH="${PYENV_ROOT}/bin:${PYENV_ROOT}/shims:${PATH}"
+    fi
```

Comment:
`VIRTUAL_ENV` is set, it seems to me pyenv will not be properly initialised? The shims would never be added to the `$PATH`?

Or am I missing something? How could pyenv work?